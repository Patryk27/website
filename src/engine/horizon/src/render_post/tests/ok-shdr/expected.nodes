Element(
    Spanned {
        inner: Element {
            name: Spanned {
                inner: "shdr",
                span: Span {
                    start: 1,
                    end: 5,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  Hello, ",
                        span: Span {
                            start: 6,
                            end: 16,
                        },
                    },
                ),
                Element(
                    Spanned {
                        inner: Element {
                            name: Spanned {
                                inner: "i",
                                span: Span {
                                    start: 17,
                                    end: 18,
                                },
                            },
                            attrs: [],
                            children: [
                                Text(
                                    Spanned {
                                        inner: "World",
                                        span: Span {
                                            start: 19,
                                            end: 24,
                                        },
                                    },
                                ),
                            ],
                        },
                        span: Span {
                            start: 16,
                            end: 28,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "!\n",
                        span: Span {
                            start: 28,
                            end: 30,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 0,
            end: 37,
        },
    },
)
Text(
    Spanned {
        inner: "\n",
        span: Span {
            start: 37,
            end: 38,
        },
    },
)