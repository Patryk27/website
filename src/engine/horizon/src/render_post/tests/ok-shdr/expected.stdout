<h3>
  Hello, <i>World</i>!
</h3>
